src/xmacro_helpers.h

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
```

Comment:
Do we really want to create an helper for macros which will be probably used only for selectors definition ?

Reply:
IMO it's utility noise that should not clutter the useful info stored in `plugins.h`